docs/openapi/schemas-pn-notification.yaml

```diff
@@ -7,3 +7,19 @@ components:
       enum:
         - PF
         - PG
+
+    paTaxId:
```

Comment:
questi due oggetti dovresti prenderli da pn-delivery cosí in caso cambiassero li hai sempre aggiornati... sono definiti in schemas-pn-notification.yaml

Reply:
giusto